web/src/components/organisms/Project/settings/index.tsx

```diff
@@ -0,0 +1,189 @@
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import Typography from "@reearth-cms/components/atoms/Typography";
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import ProjectMenu from "@reearth-cms/components/molecules/Common/projectMenu";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import TextArea from "antd/lib/input/TextArea";
+import Layout, { Header, Content } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useCallback, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import useDashboardHooks from "../../Dashboard/hooks";
+
+import useHooks from "./hooks";
+
+export interface FormValues {
+  name: string;
+  description: string;
+}
+
+const ProjectSettings: React.FC = () => {
+  const { confirm } = Modal;
+  const [form] = Form.useForm();
+  const { workspaceId } = useParams();
+  const { projectId } = useParams();
```

Comment:
Pretty sure this should work, but of course test.
```suggestion
  const { workspaceId, projectId } = useParams();
```